File: c_work/221220/ex03.py
def sfunc(num):
    num = 30 #값이 30으로 변경
    print("num = ",num)
    return num #지역변수 def sfunc(num)에 있는 값을 age에 입력


def main():
    age = 20
    age = sfunc(age) #age 20의 값을 sfunc(num = age)에 넣어준다
    print("age",age)

if __name__ == "__main__":
    main()

File: c_work/221220/ex03.c
#include <stdio.h>

int sfunc(int num){
    num = 30;
    printf("num = %d\n",num);
    return num; 
}

int main(){
    int age = 20;
    age = sfunc(age); //void sfunc(int num = age)
    printf("age = %d\n",age);
    return 0;
}